### DISCUSSION TITLE
如何调节柱状图宽度占比，0 - 1 范围数值

### DISCUSSION THREAD

**Discussion by xyr550** [antvis/G2#5549] [Q&A]
![image](https://github.com/antvis/G2/assets/73283715/917b4281-865e-4e4d-ba2e-030e6cc03019)

4.X版本有widthRatio
5.X版本如何使用哇

**Comment by pearmini** [comment]
试试如下三个配置：

```js
.scale('x', {padding, paddingInner, paddingOuter})
```
